### ISSUE TITLE
Incompatible with Include Plugin

### ISSUE THREAD

**Issue by Autumnf0x** [dokufreaks/plugin-folded#51]
Include Plugin blocks are not hided as expected.

**Comment by Floffyko** [comment]
Yes, unfortunately, the Include plug-in "escape" outside the Folded blocks. Could this be solved somehow ?

**Comment by luffah** [comment]
Using Include plugin disable all folds for me.

**Comment by GJRobert** [comment]
> Using Include plugin disable all folds for me.

But I have been using Include plugin with Folded plugin side-by-side. Greebo also.

**Comment by michitux** [comment]
Try using the `inline` flag of the include plugin inside the folded section.

**Comment by luffah** [comment]
Was using include plugin outside of folded sections.

https://www.dokuwiki.org/plugin:include#faq

But indeed `inline` flag is the solution.
```
{{page>mypageid&inline}}
```
